pretty_midi/pretty_midi.py

```diff
@@ -66,10 +68,10 @@ def __init__(self, midi_file=None, resolution=220, initial_tempo=120.):
             # Load in the MIDI data using the midi module
             if isinstance(midi_file, six.string_types) or isinstance(midi_file, pathlib.PurePath):
                 # If a string or path was given, pass it as the filename
-                midi_data = mido.MidiFile(filename=midi_file)
+                midi_data = mido.MidiFile(filename=midi_file, charset=charset)
```

Comment:
Is there a way to get mido to detect the charset of an existing file automatically, or at least get it to report the charset, or something other than having to separately specify it?

Reply:
I'm not familiar with mido. I just referred to the way to handle charsets in the mido [documentation](https://mido.readthedocs.io/en/stable/api.html?highlight=charset#mido.MidiFile). The default charset is latin1. When the user wants to specify a character set, just pass the corresponding charset parameter.